assets/js/components/legacy-notifications/notification.js

```diff
@@ -84,6 +86,12 @@ function Notification( {
 		setItem( cacheKeyDismissed, new Date(), { ttl: null } );
 
 	useMount( async () => {
+		await trackEvent(
+			`${ VIEW_CONTEXT_DASHBOARD }_site-notification`,
```

Comment:
@kuasha420 there is a mistake in IB. The event category for all events should be `${ VIEW_CONTEXT_DASHBOARD }_remote-site-notification`, see what AC says about it. Please, update all categories to use the correct name.

Reply:
@eugene-manuilov from the comments I think Felix mentions using just site-notification instead of remote-site-notification and not caring about where the notification is originated. @felixarntz, an you confirm here?

Reply:
Ah... okay, thanks @kuasha420 and @felixarntz.